src/main/java/pl/targosz/homework/Lesson8Task2/KashwayList.java

```diff
@@ -0,0 +1,77 @@
+package pl.targosz.homework.Lesson8Task2;
+
+import java.util.Arrays;
+
+public class KashwayList implements OwnList {
+
+    private Integer[] kashwayList;
+    private int maximumCapacity;
+    private int size;
+    private static final int BASE_CAPACITY = 10;
+
+    public KashwayList() {
+        this(BASE_CAPACITY);
+    }
+
+    public KashwayList(int maximumCapacity) {
+        this.maximumCapacity = maximumCapacity;
+        kashwayList = new Integer[maximumCapacity];
+        size = 0;
+    }
+
+
+    @Override
+    public int size() {
+        return kashwayList.length;
+    }
+
+    @Override
+    public boolean isEmpty() {
+        if (kashwayList.length == 0)
+            return true;
+        else
+            return false;
```

Comment:
jaki typ zwraca wyrażenie:
![image](https://user-images.githubusercontent.com/17989854/137578713-19ddee8f-7fdd-4ba3-b5b7-6052ae081d3e.png)
?

czy można to wykorzystać, aby tę metodę skrócić?